Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -192,6 +198,8 @@ private void OnUnloaded(object sender, RoutedEventArgs e)
                 _stateGroup.CurrentStateChanged -= OnVisualStateChanged;
                 _stateGroup = null;
             }
+
+            SelectionChanged += SetItemListFocusWhenNotInDetails;
```

Comment:
Should this be unsubscribing the event?

Reply:
Yes, I didn't notice the typo. I'll fix it right away, thanks!